src/high-level/src/KinDynComputations.cpp

```diff
@@ -1798,6 +1798,65 @@ SpatialMomentum KinDynComputations::getCentroidalTotalMomentum()
     return newOutputFrame_X_oldOutputFrame*base_momentum;
 }
 
+bool KinDynComputations::getCentroidalTotalMomentumJacobian(MatrixDynSize& centroidalMomentumJacobian)
+{
+    if (!getLinearAngularMomentumJacobian(centroidalMomentumJacobian))
```

Comment:
Nitpick: I typically try to avoid brace-less if statements, because a common bug is to then add another statement and forget to add braces, however this is just a suggestion.

Reply:
Done in f8bab02af15e80275f4f2c9bad59e6f1d766a458